example/ts/__relay_artifacts__/TodoApp_viewer.graphql.ts

```diff
@@ -8,6 +8,11 @@ export type TodoApp_viewer = {
     readonly " $fragmentRefs": FragmentRefs<"TodoListFooter_viewer" | "TodoList_viewer">;
     readonly " $refType": "TodoApp_viewer";
 };
+export type TodoApp_viewer$data = TodoApp_viewer;
+export type TodoApp_viewer$key = {
+    readonly " $data"?: TodoApp_viewer$data;
```

Comment:
@jstejada So are you saying that this `$data` prop definition only exists as a placeholder to return the data type from `useFragment`? I.e. nobody will ever actually pass in this key?

Reply:
> I.e. nobody will ever actually pass in this key?

if I understand your question correctly, yes, this $data is just a mechanism to extract the type and enforce the correct type is used, but this property will never exist as runtime (similar to `$fragmentRefs` or `$refType`)

Reply:
Yup, that’s what I meant. Thanks!